glide.yaml

```diff
@@ -0,0 +1,268 @@
+package: kubevirt.io/kubevirt
+ignore:
+- github.com/go-kit/kit/examples
+- github.com/googleapis/gnostic/plugins
+- github.com/go-kit/kit/log/levels
+- github.com/go-kit/kit/metrics/influxdb
+- github.com/go-kit/kit/service
+- k8s.io/apimachinery/pkg/apis/meta/v1alpha1
+- code.google.com/p/go-charset
+import:
+- package: github.com/emicklei/go-restful
+  version: 858e58f98abd32bc4d164a08e8d7ac169ae876e2
+  subpackages:
+  - log
+  - swagger
+- package: golang.org/x/net
+  version: 054b33e6527139ad5b1ec2f6232c3b175bd9a30c
+  subpackages:
+  - internal/socket
+- package: github.com/go-kit/kit
+  version: fe6fe28ba0d54b39f27e79cddba4911b7e4fffc7
+  subpackages:
+  - log
+- package: google.golang.org/grpc
+  version: d3ddb4469d5a1b949fc7a7da7c1d6a0d1b6de994
+  subpackages:
+  - metadata
+  - grpclb/grpc_lb_v1
+  - keepalive
+  - stats
+  - status
+  - tap
+- package: k8s.io/apimachinery
```

Comment:
You might be able to do this actually. The general approach I took with building glide.yaml was to execute "glide up -v" iteratively. Some deps would error out. In general what you would observe in that case is a cache-miss where a nested dependency called for a subpackage that was not originally included. In fact that's the reason for *all* the secondary dependencies listed--with the exception of go-restful if I recall. that one's actually a primary dependency.

All that said, I just tried removing apimachinery and didn't get any errors. I'm a little hesitant to remove it because the only reason I would have added it was due to cache-errors updating dependencies.